docling_ibm_models/layoutmodel/layout_predictor.py

```diff
@@ -76,34 +72,50 @@ def __init__(
         self._threshold = 0.6  # Score threshold
         self._image_size = 640
         self._size = np.asarray([[self._image_size, self._image_size]], dtype=np.int64)
-        self._use_cpu_only = use_cpu_only or ("USE_CPU_ONLY" in os.environ)
+
+        # Set device based on env var or availability
+        device_name = os.environ.get("TORCH_DEVICE", "").lower()
```

Comment:
I think we should use explicit parameters instead of generic global ENV vars. Or, in case we have no other choice, be more explicit like `DL_TORCH_DEVICE` or `DL_DEVICE`.